recipes/r-constructive/meta.yaml

```
@@ -0,0 +1,80 @@
+{% set version = '0.0.1' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-constructive
+  version: {{ version|replace("-", "_") }}
+
+source:
+
+  git_url: https://github.com/cynkra/constructive
+  git_tag: v0.0.1
```

Tags are mutable and by that insecure. Please instead use:
```suggestion
  url: https://github.com/cynkra/constructive/archive/refs/tags/v{{ version }}.tar.gz
  sha256: ...
```